code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1452,7 +1452,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		to_chat(target, "<span class='danger'>[user] fails to give you a noogie!</span>")
 		return
 
-	if(brutal_noogie)
+	if(!brutal_noogie)
```

Comment:
You shouldn't be doing `if...else` where the original condition is inverted, it's confusing.

Make it back to what it was but swap the bodies.